doc/conf.py

```diff
@@ -291,7 +291,7 @@ def setup(app):
     # Do not pop up any mayavi windows while running the
     # examples. These are very annoying since they steal the focus.
     mlab.options.offscreen = True
-except ImportError:
```

Comment:
Probably this should not be part of this PR. I seem to remember that elsewhere there was a `except Exception`since importing some libraries can raise all sorts of exceptions (that's what the comment say).

I can change this directly in master.